### PR TITLE
readme: improve language (fixed PR)

### PR DESCRIPTION
**Pull request by mara004**
## What do these changes do?

Improve the Readme.

## Are there changes in behavior for the user?

No changes in behavior of code, but users are presented an improved documentation.

## Related issue number

None

## Checklist

- [x] (I think the code is well written)
- [x] (Unit tests for the changes exist)
- [x] Documentation reflects the changes

The checklist is not really applicable as this is not a code change.

### CODE REVIEW

**Comment by mara004** [comment]
~~Same problem as in #6 again, GH doesn't add new commits after the PR was captured:
https://github.com/mara004/backports.cached_property/commit/42fed8693a097c31cc9c4b2382358b7cda04f289~~

Update: Commit appeared after some delay

File: README.rst [hunk 1]
```diff
@@ -14,51 +14,59 @@ backports.cached_property
...
+Simply do
+
 .. code-block:: python
 
     from backports.cached_property import cached_property
 
-And then python 3.8 documentation will work (because code is minimally changed):
+and it will work as documented upstream (see copy below).
 
-.. class:: cached_property
```
Comment (mara004) [inline README.rst:35]: Actually, I think this was syntactically correct, only GitHub's renderer doesn't like it (e.g. the `.. class:: cached_property` header was not displayed at all).

I changed it to make it play with GH, though this is a displeasing situation.

**Comment by mara004** [comment]
@penguinolog

**Review by penguinolog — APPROVED** [review]

**Comment by mara004** [comment]
Thanks for coming back to this!

**Pull request merged**